platforms/web/tsconfig.json

```diff
@@ -22,6 +22,7 @@
     "scripts",
     "test",
     "vite.config.ts",
-    "../../packages/common/types"
+    "../../packages/common/types",
```

Comment:
I would prefer importing them from the package instead of using a relative import here. Do you think we can do it?

Reply:
That is possible, but not in `tsconfig#includes`. We will have to create a `types/index.d.ts`, import the global definition files, and update the package.json with `"types": "./types/index.d.ts",`. Only then can we add it to the `tsconfig#compilerOptions.types` property as `@jwp/ott-common` and `@jwp/ott-ui-react`.

However, I'm intending to remove this entirely. It's better to only import the required types from common where needed.

Reply:
Do you want to do it in this PR?

Reply:
I prefer to do it separately and finish this PR asap because there are many new PRs queued...

https://github.com/Videodock/ott-web-app/pulls